pydata/models.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+from workshops.models import Person as BasePerson
+from workshops.models import Task as BaseTask
+
+
+class Person(BasePerson):
+    """ Represents a PyData speaker."""
+
+    PROFILE_REGEX = re.compile(r'^(?P<url>.+?(?=/speaker))/speaker/profile/(?P<id>[^/]+)/?')
+
+    class Meta:
```

Comment:
To encapsulate the `PROFILE_REGEX` logic within the `Person` model.
Just as `Event` model encapsulates few regexes related to the workshop URL.

I would not want them to be littered across various views.